gameapplication.py

```diff
@@ -0,0 +1,99 @@
+import time
+import re
+import IdentifyBaseWord as np
+import wordlist as wl
+
+#gameState 0 means the game is not being played
+gameState = 0
+
+
+print("Hello welcome to MediaTek's Spelling bee!")
+print("The goal of this game is to guess as many words as you can to accumulate points!")
+playGame = input("Would you like to play our game? enter yes or no: ")
+
+while (playGame != 'yes') and (playGame != 'no'):
+    print("Invalid input please enter yes or no!")
+    playGame = input("Would you like to play our game? enter yes or no\n")
```

Comment:
I think here we can change the startup message to something like: To get started, enter the -newpuzzle command for a randomly generated puzzle, or add your pangram after the command for a custom puzzle. To load a puzzle you have saved, enter the -load command with the filename. For a list of all commands and the functionality, enter -help

Just my idea though! And certainly a design decision we can make as a team.